packages/spindle-ui/src/HeroCarousel/HeroCarousel.css

```diff
@@ -0,0 +1,108 @@
+/* stylelint-disable plugin/selector-bem-pattern */
+.spui-HeroCarousel__CarouselContainer {
+  align-items: center;
+  display: flex;
+  height: 192px;
+  justify-content: center;
+  overflow: hidden;
```

Comment:
```suggestion
  overflow: hidden;
  padding: 1px 0;
  margin: -1px 0;
```
[nits] 上下のアウトラインが削れてしまっているので、例えばここでスペースを確保するなどすると良さそうです！（いくつか対応パターンはありそうです）

before | after
-- | --
<img width="343" alt="" src="https://user-images.githubusercontent.com/92707228/154386720-55a16496-e38f-43d7-b5af-797e1544533f.png"> | <img width="346" alt="" src="https://user-images.githubusercontent.com/92707228/154386711-b11d0638-79e1-4d64-b87d-d278958d9a81.png">

Reply:
:nanto_: 
修正しました！ -> https://github.com/openameba/spindle/pull/348/commits/d0c72b51d0da8dc85412cfa5495dca5df75361a9

![スクリーンショット 2022-02-17 11 34 30](https://user-images.githubusercontent.com/42470015/154393701-e6e3941c-fdf4-4fb5-b255-4daaf08aeb64.png)